tests/test_bundle.py

```diff
@@ -210,6 +211,25 @@ def _test_bundle_put(self, replica: Replica, fixtures_bucket: str):
             )
             self.assertEqual(resp_obj.json['code'], "file_missing")
 
+        # should *NOT* be able to upload a bundle containing a file with an incorrect bundle_uuid
+        # but we should get requests.codes.conflict
+        resp_obj = self.upload_file_wait(
```

Comment:
alternatively, you can just upload a new bundle with the old file, and that should fail.  that potentially saves you the extra file upload.

Reply:
fixed.